config/env/default.js

```diff
@@ -9,6 +9,9 @@ module.exports = {
   },
   port: process.env.PORT || 3000,
   host: process.env.HOST || '0.0.0.0',
+  // DOMAIN config should be set to the fully qualified application accessible
+  // URL. For example: https://www.myapp.com (including port if required).
+  domain: process.env.DOMAIN,
```

Comment:
We kind of take care of that in https://github.com/meanjs/mean/pull/1469/files/77d4a990f52b83d684a6eb73c7e019576163467e#diff-c3841fb7af244bab7b73ea4c3d72055cR64

It also makes it much complicated to default it to something because you don't know if the server is running in http or https, and this causes many more logic tests to run. It's a simple domain/url setting, I don't think we need to make it anymore complex than that.